packages/api/src/command/connected-user/get-connected-user.ts

```diff
@@ -77,3 +77,26 @@ export const getConnectedUserByTokenOrFail = async (
 
   return connectedUser;
 };
+
+export const getConnectedUsersByTokenOrFail = async (
+  provider: string,
+  token: string
+): Promise<ConnectedUser[]> => {
+  const connectedUsers = await ConnectedUser.findAll({
+    where: {
+      providerMap: {
+        [provider]: {
+          token: {
+            // TODO: Find more optimal solution
+            [Op.like]: "%" + token + "%",
```

Comment:
@RamilGaripov, why do we need this (like), instead of exact match?

Reply:
This is probably because the token has a bunch of other stuff in it, and we just want to match by a substring